src/main/java/functional/algebraic/Result.java

```diff
@@ -382,6 +409,26 @@ public void match(Consumer<? super E> ef, Consumer<? super V> vf)
         return or(convert(f));
     }
 
+    /**
+     * Perform an operation on the value inside this result, or do nothing if there is an error.
+     * @param op the operation
+     * @return the unaltered Result
+     */
+    public Result<E, V> ok(Consumer<? super V> op) {
```

Comment:
I think I'd prefer this be called `ifOk` to clarify that it performs a conditional operation. Also to prevent name collision if I change the constructor names
```suggestion
    public Result<E, V> ifOk(Consumer<? super V> op) {
```

Reply:
I'm fine with this, ok() was chosen for symmetry with Rust though.

Reply:
Doesn't rust's `ok()` method just return an `Option<V>`? The version of the docs I'm looking at indicates that

Reply:
Oh, it does too! Sounds good.